imap_processing/lo/l0/lol0.py

```diff
@@ -0,0 +1,48 @@
+"""General Lo L0 data class used for parsing data and setting attributes."""
+from dataclasses import dataclass, fields
+
+from imap_processing.ccsds.ccsds_data import CcsdsData
+
+
+@dataclass
+class LoL0:
+    """Data structure for common values across histogram and direct events data.
+
+    Attributes
+    ----------
+    ground_sw_version : str
+        Ground software version
+    packet_file_name : str
+        File name of the source packet
+    ccsds_header : CcsdsData
+        CCSDS header data
+    """
+
+    ground_sw_version: str
+    packet_file_name: str
```

Comment:
Is this something Lo wants in their datafiles? Because otherwise, it's not really needed.

Reply:
They're not asking for it in their data but I thought we were adding that to the cdf headers. I might be wrong about that though.

Reply:
I don't think so, but I am not sure. I have it in GLOWS because they specifically asked for it in their algorithm document.